bands/models.py

```diff
@@ -133,8 +135,8 @@ def w9_extension(self):
         return extension(self.w9_filename)
 
     @property
-    def completed(self):
-        return self.w9_filename
+    def status(self):
+        return 'Yes <a href="{}">(view file)</a>'.format(self.w9_url) if self.w9_filename else ''
```

Comment:
We probably want to avoid putting html in our database models. Would be nice if we could move this to a template, or a macro or something.